webpack.config.js

```diff
@@ -28,6 +28,11 @@ module.exports = function (env) {
                 },
 
                 {
+                    test: /\.json$/,
```

Comment:
json-loader is a part of Webpack 2 and works out of the box. Read documentation more carefully https://github.com/webpack-contrib/json-loader